py-polars/Cargo.toml

```diff
@@ -158,6 +158,9 @@ features = [
   "string_justify",
   "arg_where",
   "date_offset",
+  "aws",
```

Comment:
Can we move these to the `all` feature. They are not needed as default.

Reply:
Thanks for the info, moved.